tests/Unit/Repositories/RepositoryTestCase.php

```diff
@@ -22,22 +23,56 @@ protected function createMany(int $count = 3): iterable
         return factory($this->repo->modelClass(), $count)->create();
     }
 
+    /** @test */
+    public function itCanCreateAModel()
+    {
+        $model = $this->create();
+
+        $this->assertNotNull($model, 'Model IS null');
+    }
+
+    /** @test */
+    public function itCanListAllTheModels()
+    {
+        $n = 3;
+
+        $this->createMany($n);
```

Comment:
the default for this method is to create 3 models, no need to pass that argument

Reply:
I did that so that if the default ever changes this test won't break. It makes it less brittle.

But this is even slicker:

```php
$this->assertCount($this->createMany()->count(), $this->repo->list());
```

Reply:
I see, However the default should never change as if a test requires a different value there they just pass it in.

I do like the new proposed assert above. Clean.